recipes/server.rb

```diff
@@ -17,15 +17,24 @@
 # limitations under the License.
 #
 
+Chef::Recipe.send(:include, MariaDB::Helper)
 case node['mariadb']['install']['type']
 when 'package'
-  include_recipe "#{cookbook_name}::repository"
+  if use_os_native_package?(
+      node['mariadb']['install']['prefer_os_package'],
+      node['platform'], node['platform_version'])
+    # currently, no releases with apt (e.g. ubuntu) ship mariadb
+    # only provide one type of server here (with yum support)
+    include_recipe "#{cookbook_name}::_redhat_server_native"
```

Comment:
Yeah, I agree that we need to provide native package support later for debian like platforms as well. That's why I put two lines of comments before calling the include_recipe only for redhat. Otherwise, maybe we need to add an empty case here with TODO. But it looks kind of odd here, because no debian like releases actually do that now.